api/v1/inference_server.proto

```diff
@@ -28,25 +21,29 @@ message CreateChatCompletionRequest {
       Function function = 3;
     }
 
-    // Refer to https://platform.openai.com/docs/api-reference/chat/create#chat-create-messages
-    message ImageUrl {
-      string url = 1;
-      string detail = 2;
-    }
-    message InputAudio {
-      string data = 1;
-      string format = 2;
-    }
     message Content {
-      ContentType type = 1;
+      // Refer to https://platform.openai.com/docs/api-reference/chat/create#chat-create-messages
+      message ImageUrl {
+	string url = 1;
+	string detail = 2;
+      }
+      message InputAudio {
+	string data = 1;
+	string format = 2;
+      }
```

Comment:
nits: it seems like the indent is off.

Reply:
ah confusing, but I've moved these messages inside `Content` to follow the pattern in this file.